### PR TITLE
GCI 2018: Added certificate validation

### PR DESCRIPTION
**Pull request by bartekpacia**
# Overview
I wasn't sure how to do this task, so decided to create a separate component for creating + validation of the certificate. It's my 2nd angular task but I tried to do my best 💪 

It performs some very basic validaton, and if it's successful user can proceed.

I've used official Angular docs for teching myself how to do it.

# Task
[Adding Validations to the Template](https://codein.withgoogle.com/dashboard/task-instances/5728661895380992/)

# Graphics
validation-cert-gen-front

### CODE REVIEW

**Comment by bartekpacia** [comment]
Do you mean the bug that user can "Preview" without even filling the required fields?
I was so tired that I forgot about this. Fixing it today

**Comment by bartekpacia** [comment]
Now, user can't proceed if form isn't filled correctly.

File: src/app/certificate-form/certificate-form.component.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+  constructor() { }
+
+  ngOnInit() {
+  }
+
+  certificate_data = new CertificateData(null, null, null, null);
+
+  preview() {
+    window.location.href = "http://localhost:4200/basic";
```
Comment (nikitadhiman) [inline src/app/certificate-form/certificate-form.component.ts]: no url hard code please... use routers

Comment (bartekpacia) [reply src/app/certificate-form/certificate-form.component.ts]: Fixed :)

**Comment by bartekpacia** [comment]
Build's passing!

**Comment by nikitadhiman** [comment]
Thanks for your work... closing this for now

**Pull request closed without merging**